src/styles/main.scss

```diff
@@ -10,77 +10,160 @@ body {
   font-weight: 900;
 }
 
+// .field-cell {
+  // background: #d6cdc4;
+  // width: 75px;
+  // height: 75px;
+  // border-radius: 5px;
+  // color: #776e65;
+  // box-sizing: border-box;
+  // text-align: center;
+  // vertical-align: center;
+  // user-select: none;
+
+  // &--2 {
+  //   background: #eee4da;
+  // }
+
+  // &--4 {
+  //   background: #ede0c8;
+  // }
+
+  // &--8 {
+  //   background: #f2b179;
+  //   color: #f9f6f2;
+  // }
+
+  // &--16 {
+  //   background: #f59563;
+  //   color: #f9f6f2;
+  // }
+
+  // &--32 {
+  //   background: #f67c5f;
+  //   color: #f9f6f2;
+  // }
+
+  // &--64 {
+  //   background: #f65e3b;
+  //   color: #f9f6f2;
+  // }
+
+  // &--128 {
+  //   background: #edcf72;
+  //   color: #f9f6f2;
+  // }
+
+  // &--256 {
+  //   background: #edcc61;
+  //   color: #f9f6f2;
+  // }
+
+  // &--512 {
+  //   background: #edc850;
+  //   color: #f9f6f2;
+  // }
+
+  // &--1024 {
+  //   background: #edc53f;
+  //   color: #f9f6f2;
+  // }
+
+  // &--2048 {
```

Comment:
don't leave comments

Reply:
that also leads to lint errors:
![image](https://user-images.githubusercontent.com/86923089/212533586-23c872f0-8716-4d8d-9e38-c044c09e25ef.png)